jsapp/js/components/modalForms/formMedia.es6

```diff
@@ -64,6 +84,20 @@ class FormMedia extends React.Component {
     });
   }
 
+  uploadMedia(formMediaJSON) {
+    // Reset error message before uploading again
+    this.setState({fieldsErrors: {}});
+    const callbacks = {
+      onComplete: this.onGetMediaCompleted.bind(this),
+      onFail: this.onUploadFailed.bind(this)
+    }
+    actions.media.uploadMedia(this.props.asset.uid, formMediaJSON, callbacks);
```

Comment:
Similarly to above - remove `callbacks` and add listeners to `componentDidMount`:

```
actions.media.uploadMedia.completed.listen(this.onUploadCompleted);
actions.media.uploadMedia.failed.listen(this.onUploadFailed);
```